src/state/wallet/hooks.ts

```diff
@@ -101,15 +101,16 @@ export function useTokenBalancesWithLoadingIndicator(
   const chainId = customChain || currentChain
 
   const tokens = useMemo(() => {
-    return tokenParams?.[0]?.chainId === chainId ? tokenParams : EMPTY_ARRAY
+    return tokenParams?.[0]?.chainId === chainId ? tokenParams : []
   }, [tokenParams, chainId])
 
   const isFetchOtherChain = chainId !== currentChain
 
-  const balancesCurrentChain = useTokensBalance(isFetchOtherChain ? EMPTY_ARRAY : tokens, chainId)
+  const balancesCurrentChain = useTokensBalance(isFetchOtherChain ? [] : tokens, chainId)
+
   const [balancesOtherChain, isLoadingAnotherChain] = useTokensBalanceOfAnotherChain(
     chainId,
-    isFetchOtherChain ? tokens : EMPTY_ARRAY,
+    isFetchOtherChain ? tokens : [],
```

Comment:
tell us why

Reply:
Actually I don't know why we need a const like that while we can just use []

Reply:
it can make a loop if pass [] directly instead of EMPTY_ARRAY
useEffect(()=>{

},[ dependencies])

Reply:
because it will create a new array when every render

Reply:
Ah okay, let's me think

Reply:
Reverted